docs/messages.md

```diff
@@ -0,0 +1,50 @@
+title: Messages
+description: A superclass for the messages of the bitcoin network
+---
+# Messages
+
+The bitcoin protocol specifies a set of [messages](https://en.bitcoin.it/wiki/Protocol_specification) that can be sent from peer to peer. `bitcore-p2p` provides partial support for some of these messages.
```

Comment:
"provides partial support for some of these messages." sounds like only some messages are *incompletely* supported. I would change it to "provides support for some of these messages."